src/test/scala/com/fulcrumgenomics/sv/tools/AggregateSvPileupToBedPETest.scala

```diff
@@ -108,7 +108,7 @@ class AggregateSvPileupToBedPETest extends UnitSpec {
 
     new AggregateSvPileupToBedPE(input = input, output = output).execute()
 
-    val fields: Seq[String] = classOf[BedPE].getDeclaredFields.map(_.getName).toIndexedSeq
+    val fields: Seq[String] = test_bed_pe.names
```

Comment:
There are synthetic fields contaminating the declared fields list since fgbio 3.0.0. It could be better solved like:

```scala
val fields: Seq[String] = new ReflectiveBuilder(classOf[BedPE]).argumentLookup.ordered.map(_.name)